packages-exp/auth-exp/src/platform_browser/persistence/indexed_db.ts

```diff
@@ -166,6 +166,7 @@ function deleteObject(db: IDBDatabase, key: string): Promise<void> {
 export const _POLLING_INTERVAL_MS = 800;
 
 class IndexedDBLocalPersistence implements Persistence {
+  private static readonly TRANSACTION_RETRY_COUNT = 3;
```

Comment:
Don't use static variables (per Fei's tree shaking work)

Use this as a file-local `const` instead

Reply:
done